tests/test_multi_agent.py

```
@@ -6,10 +6,13 @@
 from graphrag_sdk.relation import Relation
 from graphrag_sdk.ontology import Ontology
 from graphrag_sdk.agents.kg_agent import KGAgent
+from graphrag_sdk.models.litellm import LiteModel
 from graphrag_sdk.orchestrator import Orchestrator
 from graphrag_sdk.attribute import Attribute, AttributeType
-from graphrag_sdk.models.gemini import GeminiGenerativeModel
-from graphrag_sdk import KnowledgeGraph, KnowledgeGraphModelConfig
+from graphrag_sdk import (KnowledgeGraph,
+                          KnowledgeGraphModelConfig,
+                          GenerativeModelConfig)
```

Ruff (0.11.9)

`graphrag_sdk.GenerativeModelConfig` imported but unused

Remove unused import: `graphrag_sdk.GenerativeModelConfig`

(F401)